gbfs.md

```diff
@@ -576,10 +578,83 @@ Field Name | Required | Type | Defines
 \-&nbsp;`url` | Optional | URL | URL where the customer can learn more about this pricing plan.
 \-&nbsp;`name` | Yes | String | Name of this pricing plan.
 \-&nbsp;`currency` | Yes | String | Currency used to pay the fare. <br /><br /> This pricing is in ISO 4217 code: http://en.wikipedia.org/wiki/ISO_4217 <br />(e.g. `CAD` for Canadian dollars, `EUR` for euros, or `JPY` for Japanese yen.)
-\-&nbsp;`price` | Yes | Non-negative float OR String | Fare price, in the unit specified by currency. If String, must be in decimal monetary value.
+\-&nbsp;`price` | Yes | Non-Negative float OR String | Fare price, in the unit specified by currency. If string must be in decimal monetary value. Note: v3.0 will only allow non-negative float, therefore future implementations must be non-negative float.<br /><br />In case of non-rate price, this field is the total price. In case of rate price, this field is the base price that is charged only once per trip (e.g., price for unlocking) in addition to `per_km_pricing` and/or `per_min_pricing`.
```

Comment:
For the description of `price`, it could also be the base price that is charged to access the vehicles for a certain period of time (instead of being per trip). For example, a flat fee to have access to bikes for 24 hours, which is not charged per trip but once at the start of the 24h period, and then each trip can be charged per min or per km within these 24 hours.

Reply:
Currently, this MVP is designed for modelling one-way trips, so passes are not easily modelled this way, however it can be done using two concurrent rates (one with an interval of 1440 minutes). We can add more specific language about `price` acting as an unlock fee, and once this is in the spec we will be sure to add a handful of examples to show how it can be used in these various ways. Hope that helps!

Reply:
OK I see, thanks!